src/client/js/partials/settings.js

```diff
@@ -44,6 +47,7 @@ if (settingsAlertOptionsInputs?.length) {
         throw new Error(`Updating communication option failed: ${err}`)
       }
       event.preventDefault()
+      window.gtag('event', 'Change Email Preference', { action: 'click', page_location: location.href })
```

Comment:
Question: Are button clicks not auto-tracked for free in GA4?

Reply:
After testing, I don't think so... it looks like only outbound links are tracked by default with our current config. With GTM this is a bit different. We could have a generic `click` handler we always install, but I thought we should try to measure things more explicitly. We also don't want the generic `click` event since you'd have to drill down in the reports to see where it came from.